packages/web-components/src/index.html

```diff
@@ -18,6 +18,18 @@
 
   </head>
   <body>
-    <my-component first="Stencil" last="'Don't call me a framework' JS"></my-component>
+    <va-table uswds table-title="a cool table">
```

Comment:
Was this just for testing?

Reply:
@jamigibbs - it slipped through - I always try to check that I don't commit that file!